docs/usage.md

```diff
@@ -103,58 +102,68 @@ jrnl -starred
+displays all entries in which either `@pinkie` or `@WorldDomination`
 occurred.
 
 ```sh
 jrnl -n 5 -and @pinkie @WorldDomination
 ```
 
-the last five entries containing both `@pinkie` **and** `@worldDomination`.
-You can change which symbols you'd like to use for tagging in the
-configuration.
+displays the last five entries containing both `@pinkie` **and**
+`@worldDomination`. You can change which symbols you'd like to use for tagging
+in the configuration.
 
 !!! note
-    `jrnl @pinkie @WorldDomination` will switch to viewing mode because
-    although **no** command line arguments are given, all the input strings
-    look like tags - _jrnl_ will assume you want to filter by tag.
+    Entering `jrnl @pinkie @WorldDomination` will display entries in which both
+    tags are present because, although no command line arguments are given, all
+    of the input strings look like tags. `jrnl` will assume you want to filter
+    by tag, rather than create a new entry that consists only of tags.
+
 
-## Editing older entries
+## Editing Existing Entries
 
-You can edit selected entries after you wrote them. This is particularly
-useful when your journal file is encrypted. To use this feature, you need
-to have an editor configured in your journal configuration file (see
-`advanced usage <advanced>`)
+You can edit entries after writing them. This is particularly useful when your
+journal file is encrypted. To use this feature, you need to have an external
+editor configured in your configuration file (see `advanced usage <advanced>`).
+You can also edit only the entries that match specific search criteria. For
+example,
 
 ```sh
 jrnl -until 1950 @texas -and @history --edit
 ```
 
-Will open your editor with all entries tagged with `@texas` and
-`@history` before 1950. You can make any changes to them you want; after
-you save the file and close the editor, your journal will be updated.
+opens your external editor displaying all entries tagged with `@texas` and
+`@history` that were written before 1950. After making changes, save and close
+the file, and only those entries will be modified (and encrypted, if
+applicable).
+
+If you are using multiple journals, it's easy to edit specific entries from
+specific journals. Simply prefix the filter string with the name of the journal.
+For example,
+
+```sh
+jrnl work -n 1 --edit
+```
 
-Of course, if you are using multiple journals, you can also edit e.g.
-the latest entry of your work journal with `jrnl work -n 1 --edit`. In
-any case, this will bring up your editor and save (and, if applicable,
-encrypt) your edited journal after you save and exit the editor.
+opens the most recent entry in the 'work' journal in your external editor.
 
-You can also use this feature for deleting entries from your journal
+You can also use this feature for deleting entries from your journal. Open an external editor with the entries you want to delete...
```

Comment:
**Optional:**

We actually have a `--delete` command now (was just released with `v2.4`). If you could mention that here, it would be great.

It's also ok if not; we can add it later.

Reply:
Back in the day there was an example about deleting all entries related to an ex. I ended up needing to do that (bummer), but instead of deleting the entries, I moved them into another journal. Is that something that could be automated? Something like `jrnl -s 'ex' > ~/ex.md && jrnl -s 'ex' --delete`? Maybe it could be `jrnl -s 'string' --move -o ~/string.md` -- just spitballing.

Reply:
...oh yeah, and more to the point, I'll add the reference to the --delete command. Are there other new things that you'd like me to add while I'm at it?

Reply:
It looks like `jrnl -contains` also isn't documented. I think including that would cover everything.

As for moving entries, feel free to create a feature request issue if you're interested in the feature. Alternatively, if you want to spell out a recipe for that, the recipes section of the documentation could be a good place for this.

Reply:
`-contains` is just a search for a specific string, right? By the way, as of a few years ago I was worried that `jrnl` was abandonware. When did all of this new activity start?

Reply:
Yes, that's correct, `-contains` is a case-insensitive string search.

As for the activity, @wren and I noticed that this project was running into some issues a year ago, and after some chatting with the project's founder, we joined as maintainers. Once we got a regular routine along with some resolutions to compatibility and build issues, the community showed up with a bunch of new features like `-contains` and `-delete`.

Reply:
@guydebros Thank you for all the hard work on this; it's looking great. 

For your idea about a move command, can you please put into a new ticket, so we can discuss the pros/cons of the proposal?